### ISSUE TITLE
Passing additional key/value with all events sent to the Jitsu Hosted Tracker

### ISSUE THREAD

**Issue by azhard4int** [jitsucom/jitsu#334]
### Problem

What problem this feature is going to solve. Why this problem is important

We are experimenting an  in-house application on top of Jitsu where we would like to add a project_id with each event passed through the Jitsu SDK. Right now, we have to pass this value in each of our event. An example of such event is:

```
eventN.track('pageview', {'project_id': 'uuid-key'})
```

### Solution

Proposed solution: architecture, implementation details etc

It would be nice to have an option where we can add initiate a project id when we are adding a script in `<head>` section of the page. An example could be:

```
eventN.init('project_id', 'uuid-value') // this get passed along with each event that is bent sent to hosted tracker.
```

To make it more generic, it would be nice to have an option.

```
eventN('set', 'project_id', 'uuid-value');
jitsu('set', 'project_id', 'uuid-value');
```

**Comment by vklimontovich** [comment]
Hi @azhard4int! Thank you for submitting the issue. That's definitely something we can implement fairly quickly.

I have a question about persistence: do you think those settings should be persisted between sessions (in cookie / local storage)?

**Comment by azhard4int** [comment]
@vklimontovich Yes, the information should be persisted between sessions. On the first time when the value is being set, it should be stored in a cookie/local storage. After that the same key should be used and passed with the events that are emitting.

**Comment by vklimontovich** [comment]
Hi @azhard4int !

This has been implemented. Please note, that the feature is available only in [JS SDK 2.0](https://jitsu.com/docs/sending-data/js-sdk) (see upgrade instruction if you're using [the previous version](https://jitsu.com/docs/sending-data/js-sdk/migrating))

If you're using HTML snippet, here's a [relevant documentation](http://localhost:3000/docs/sending-data/js-sdk/snippet#persistent-properties)
